tests/e2e/vsphere_volume_expansion.go

```diff
@@ -2204,3 +2760,55 @@ func sizeInMb(size resource.Quantity) int64 {
 	actualSize = actualSize / (1024 * 1024)
 	return actualSize
 }
+
+func staticProvisioningPreSetUpUtil(ctx context.Context, f *framework.Framework, c clientset.Interface, storagePolicyName string) (*restclient.Config, *storagev1.StorageClass, string) {
+	log := logger.GetLogger(ctx)
+
+	namespace := getNamespaceToRunTests(f)
+	// Get a config to talk to the apiserver
+	k8senv := GetAndExpectStringEnvVar("KUBECONFIG")
+	restConfig, err := clientcmd.BuildConfigFromFlags("", k8senv)
+	gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+	ginkgo.By("Get storage Policy")
```

Comment:
Combine this line and next line into one statement ?

Reply:
Done